internal/cli/command/cluster/install_secret.go

```diff
@@ -0,0 +1,206 @@
+// Copyright © 2019 Banzai Cloud
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cluster
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"strings"
+
+	"github.com/banzaicloud/banzai-cli/.gen/pipeline"
+	"github.com/banzaicloud/banzai-cli/internal/cli"
+	"github.com/banzaicloud/banzai-cli/internal/cli/input"
+	"github.com/goph/emperror"
+	"github.com/pkg/errors"
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+	"gopkg.in/AlecAivazis/survey.v1"
+)
+
+type installSecretOptions struct {
+	file  string
+	merge bool
+}
+
+func NewSecretInstallCommand(banzaiCli cli.Cli) *cobra.Command {
+	options := installSecretOptions{}
+
+	cmd := &cobra.Command{
+		Use:     "secret SECRET_NAME CLUSTER_NAME",
```

Comment:
I would consider using an option for cluster info:

`--cluster-id 1234` or `--cluster-name asd`